_data/mu-libraries.yml

```diff
@@ -0,0 +1,25 @@
+content:
+    - name: Mu-Scala
+      description: Mu provides the ability to combine RPC protocols, services, and clients in your Scala program, thanks to gRPC.
+      icon: img/mu-scala.svg
+      colour: "#F44336"
+      links:
+        website: https://higherkindness.io/mu-scala
+        repository: https://github.com/higherkindness/mu-scala
+
+    - name: Mu-Kotlin
+      description: Mu-Kotlin is an incoming purely functional library for building microservices in Kotlin.
+      icon: img/mu-kotlin.svg
+      colour: "#0495D7"
+      links:
+        website: https://higherkindness.io/mu-kotlin
+        repository: https://github.com/higherkindness/mu-kotlin
+
+    - name: Mu-Haskell
+      description: A set of packages that helps you build both servers and clients for microservices, allowing you to focus on your domain logic.
```

Comment:
I think we should align the test, but it's something we can do it in a following PR